### DISCUSSION TITLE
How To Recover Overwritten Auto Save State?

### DISCUSSION THREAD

**Discussion by bLaCkwEw** [OnionUI/Onion#1713] [Support]
I was trying to learn the hotkeys of Onion OS and accidentally rolled back to last save in my Pokemon Fire Red playthrough. 

Problem is, I've been using the auto save state feature as the main save, so when it rolled back, I basically went from about 15h into the game, to about 3h-4h. 

Is there a way to recover that save? 
Is it by any chance saved on the sd card? 
Can I potentially use a "deleted files recovery" utility on Linux/Windows to try to recover the file?
Is there a way to make auto saves permanent going further?

**Comment by jlnr** [comment]
My daughter is also playing a lot Fire Red as her first game, so she is not used to actively saving the game. OnionOS would be lot more "fire and forget" if it had an automatically rotating list of save states. Manually incrementing and decrementing the save slot number feels a bit out of place because it is such a low-level operation.

**Reply by Aemiii91** [reply]
Onion by default auto increments the save slot, and only keeps 10 - i.e. an automatically rotating list of save states 🤨
